component/portal/src/main/java/org/exoplatform/portal/mop/rest/EntityBuilder.java

```diff
@@ -98,9 +99,66 @@ public static List<UserNodeRestEntity> toUserNodeRestEntity(Collection<UserNode>
           resultNode.setPageAccessPermissions(accessPermissions);
         }
       }
-      resultNode.setChildren(toUserNodeRestEntity(userNode.getChildren(), expand, organizationService, layoutService, userACL));
+      if (expandBreadcrumb) {
+        List<UserNodeBreadcrumbItem> userNodeBreadcrumbItemList = getUserNodeBreadcrumbList(userPortal, userNode);
+        Collections.reverse(userNodeBreadcrumbItemList);
+        resultNode.setUserNodeBreadcrumbItemList(userNodeBreadcrumbItemList);
+      }
+
+
+      resultNode.setChildren(toUserNodeRestEntity(userNode.getChildren(),
+              expand,
+              organizationService,
+              layoutService,
+              userACL,
+              userPortal,
+              expandBreadcrumb));
```

Comment:
```suggestion
              false));
```
we don't need to expand breadcrumb for children